api/faers/api.js

```diff
@@ -707,6 +708,13 @@ TrySearch = function(index, params, es_search_params, response) {
         }
         response_json.results.push(result);
       }
+      var nextSkip = params.skip + params.limit;
+      if (body.hits.total > nextSkip) {
+        request.query.skip = nextSkip;
+        var nextQuery = qs.stringify(request.query);
```

Comment:
Please note these two lines which are very important. This modifies the provided query (the part after `?` in a URL) providing the required skip.

Instead of merging this PR you have implemented your own version which doesn't work.

Reply:
Actually it seems that search is carried over but API is not.